core/tests/get_credit_card_test.rs

```diff
@@ -0,0 +1,52 @@
+#[cfg(test)]
+mod get_credit_card_test {
+    use lockbook_core::assert_matches;
+    use lockbook_core::service::api_service;
+    use lockbook_core::service::api_service::ApiError;
+    use lockbook_core::service::test_utils::{
+        generate_account, generate_monthly_account_tier, generate_root_metadata, test_credit_cards,
+    };
+    use lockbook_models::api::*;
+
+    #[test]
+    fn get_credit_card() {
+        let account = generate_account();
+        let (root, _) = generate_root_metadata(&account);
+
```

Comment:
In the tests you create (here in `get_credit_card_test` as well as in `switch_account_tier_test`), you init an account and root and then do a NewAccountRequest. 3/7 times have a blank line before the new account request is made, the other 4 times you don't. I'd say just opt for one style for all 7.